README.md

```diff
@@ -52,28 +52,28 @@ You should ideally provide a config file for your specific sensor, but default o
 
 Supported models, where sensor_model is the ROS param to be used at launch:
 
-| Manufacturer | Model         | sensor_model | Configuration file | Test status     |
-|--------------| ------------- | ------------ | ------------------ | --------------- |
-| Hesai        | Pandar 64     | Pandar64     | Pandar64.yaml      | Complete        |
-| Hesai        | Pandar 40P    | Pandar40P    |                    | Complete        |
-| Hesai        | Pandar XT32   | PandarXT32   | PandarXT32.yaml    | Complete        |
-| Hesai        | Pandar XT32M  | PandarXT32M  | PandarXT32M.yaml   | Limited         |
-| Hesai        | Pandar QT64   | PandarQT64   |                    | Limited         |
-| Hesai        | Pandar QT128  | PandarQT128  | PandarQT128.yaml   | Limited         |
-| Hesai        | Pandar AT128  | PandarAT128  | PandarAT128.yaml   | Complete        |
-| Hesai        | Pandar 128E4X | Pandar128E4X |                    | Limited         |
-| Velodyne     | VLP-16        | VLP16        | VLP16.yaml         | Limited         |
-| Velodyne     | VLP-16-HiRes  | VLP16        |                    | Limited         |
-| Velodyne     | VLP-32        | VLP32        | VLP32.yaml         | Limited         |
-| Velodyne     | VLS-128       | VLS128       | VLS128.yaml        | Limited         |
+| Manufacturer | Model         | sensor_model | Configuration file | Test status |
```

Comment:
[nits] how about using emoji ✅ for describing test status?

Reply:
Added in https://github.com/tier4/nebula/pull/28/commits/5e82c3b9d9b91382efdbdaa598bed7bbe72e01d1